services/web/client/source/resource/qxapp/node-meta-v0.0.1.json

```diff
@@ -0,0 +1,327 @@
+                                        "structure"
+                                    ],
+                                    "properties": {
+                                        "type": {
+                                            "description": "type of the property",
+                                            "type": "string",
+                                            "enum": ["SelectBox"]
+                                        },
+                                        "structure": {
+                                            "type": "array",
+                                            "minItems": 1,
+                                            "items": {
+                                                "type":"object",
+                                                "additionalProperties": false,
+                                                "required": [
+                                                    "key",
+                                                    "label"
+                                                ],
+                                                "properties": {
+                                                    "key": {
+                                                        "type": ["string","boolean","number"]
+                                                    },
+                                                    "label": {
+                                                        "type": "string"
+                                                    }
+                                                },
+                                                "examples": [
+                                                    [
+                                                        { "key": "rat", "label": "The Rat"},
+                                                        { "key": "dog", "label": "Bello the Dog"}
+                                                    ]
+                                                ]
+                                            }
+                                        }
+                                    }
+                                }
+                            ]
+                        }
+                    }
+                }
+            }
+        },
+        "outputs": {
+            "type": "object",
+            "description": "definition of the outputs of this node",
+            "patternProperties": {
+                "^[-_a-zA-Z0-9]+$": {
+                    "type": "object",
+                    "description": "all the output produced by this node",
+                    "additionalProperties": false,
+                    "required": [
+                        "displayOrder",
+                        "label",
+                        "description",
+                        "type"
+                    ],
+                    "properties": {
+                        "displayOrder": {
```

Comment:
Use the same definitions as for inputs? duplication...
(same for label, description, type, filetokeymap)

Reply:
yes, there might be some simplification possible using references ...